.evergreen/config.yml

```diff
@@ -200,6 +208,30 @@ tasks:
         vars:
           TS_VERSION: "next"
           TRY_COMPILING_LIBRARY: "false"
+  - name: run-benchmarks-node
+    commands:
+      - func: fetch source
+        vars:
+          NODE_LTS_VERSION: v16.20.1
```

Comment:
Our perf nodejs version is v18.16.0 in the driver, should we use the same here?

Reply:
Not sure why I thought we were using 16 for the perf distribution. Thanks for catching that.